packages/diagrams/frontend/sirius-components-diagrams/src/renderer/palette/quick-access-tool/PaletteQuickAccessToolBar.tsx

```diff
@@ -92,6 +93,12 @@ export const PaletteQuickAccessToolBar = ({
       );
     }
 
+    quickAccessToolComponents.push(
+      <ResetManuallyLayoutedHandlesTool
+        diagramElementId={diagramElementId}
+        key={'tool_resetManuallyLayoutedHandlesTool'}></ResetManuallyLayoutedHandlesTool>
```

Comment:
Why a key is necessary here? You can also close the element directly since it won't have any children